build.sbt

```diff
@@ -1,6 +1,6 @@
 import sbtassembly.MergeStrategy
 
-val prinzV = "1.2.0-preview-staging"
+val prinzV = "1.2.1-preview-staging"
```

Comment:
Please change to  1.2.0-preview-staging-2021-12-21 to make this update and we will keep this convention in future if needed